polynomial/__init__.py

```diff
@@ -128,9 +142,13 @@ def __setitem__(self, degree, new_value):
 
     def __iter__(self):
         """Return the coefficients from the highest degree to the lowest."""
-        return iter(reversed(self._vector))
+        return reversed(self._vector)
 
     def __repr__(self):
+        terms = ', '.join([str(ak) for ak in self._vector][::-1])
+        return "Polynomial({0})".format(terms)
```

Comment:
I fixed this in the other PR but can you please provide a docstring for consistency's sake like 
``` python
"""Return repr(self)."""
```

Reply:
I think this would fix the merge conflicts now as well.